namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala

```diff
@@ -143,12 +151,14 @@ class HttpControlService(storage: DtabStore, namers: Ns => NameInterpreter)
       handlePostDtab(ns, req)
     case DtabUri(Method.Delete, Some(ns)) =>
       handleDeleteDtab(ns)
-    case BindUri(ns, path) =>
+    case BindUri(Some(ns), path) =>
       handleGetBind(ns, path)
-    case AddrUri(ns, path) =>
+    case AddrUri(Some(ns), path) =>
       handleGetAddr(ns, path)
-    case DelegateUri(ns, path) =>
+    case DelegateUri(Some(ns), path) =>
       handleGetDelegate(ns, path)
+    case DelegateUri(None, path) =>
+      delegateApiHander(req)
```

Comment:
fwiw, i expect that this is going to break in an upcoming change.  I expect it will work as follows:
- / is a valid (root) namespace.
- all delegation requests will accept a dtab (override) that is appended to the namespace's dtab.
- a special namespace, e.g. _/.empty_, will be supported to support requests that don't use an underlying dtab.